sdk/storage_blobs/src/blob/operations/clear_page.rs

```diff
@@ -10,9 +10,9 @@ operation! {
     ClearPage,
     client: BlobClient,
     ba512_range: BA512Range,
-    ?sequence_number_condition: SequenceNumberCondition,
-    ?if_modified_since_condition: IfModifiedSinceCondition,
-    ?if_match_condition: IfMatchCondition,
+    ?if_sequence_number: SequenceNumberCondition,
```

Comment:
Should we change `SequenceNumberCondition` to `IfSequenceNumber`?

Reply:
Opened PR